install_config/aggregate_logging.adoc

```diff
@@ -102,308 +88,409 @@ deployed. To control component placement, specify node selectors per component t
 be applied to their deployment configurations.
 ====
 
-. Create the logging xref:../admin_guide/service_accounts.adoc#admin-guide-service-accounts[service
-accounts] and custom roles:
-+
-----
-$ oc new-app logging-deployer-account-template
-----
-+
+[[aggregate-logging-ansible-variables]]
+== Specifying Ansible Variables
+
+Parameters for the EFK deployment may be specified to the
+ xref:../install/advanced_install.adoc#configuring-ansible[inventory host file]
+to override the
+ https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_metrics/defaults/main.yaml[defaults].  Read the
+xref:..aggregate_logging.adoc#aggregated-elasticsearch[Elasticsearch]
+and the xref:..aggregate_logging.adoc#aggregated-fluentd[Fluentd] sections
+before choosing parameters :
+
 [NOTE]
 ====
-If you deployed logging previously, for example in a different project, then
-it is normal for the cluster roles to fail to be created because they already exist.
+Ensure that port 9300 is open. By default the Elasticsearch service
+uses port 9300 for TCP communication between nodes in a cluster.
```

Comment:
I realize this is just moved from elsewhere, but... this doesn't make any sense. Make sure port 9300 is open on what? It's a port on a pod IP, it's up to kubernetes to ensure that it's open there.